geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/mapreduce/kde/compare/ComparisonAccumuloStatsReducer.java

```diff
@@ -109,7 +109,7 @@ protected void reduce(
 
 	private Point2d[] fromIndexToLL_UR(
 			final long index ) {
-		final double llLon = ((Math.floor(index / numYPosts) * 360.0) / numXPosts) - 180.0;
+		final double llLon = ((Math.floor(index / (double) numYPosts) * 360.0) / numXPosts) - 180.0;
```

Comment:
double check here, but since it's in a Math.floor it's being implicitly converter to a double regardless  (and for int/int there's no need to Math.floor()) - so I assume this is what was actually intended here.